src/components/HTMLEngineProvider/HTMLRenderers/AnchorRenderer.js

```diff
@@ -61,8 +61,9 @@ function AnchorRenderer(props) {
             style={{...props.style, ...parentStyle, ...styles.textUnderlinePositionUnder, ...styles.textDecorationSkipInkNone}}
             key={props.key}
             displayName={displayName}
+
             // Only pass the press handler for internal links. For public links or whitelisted internal links fallback to default link handling
-            onPress={internalNewExpensifyPath || internalExpensifyPath ? Link.openLink : undefined}
+            onPress={(internalNewExpensifyPath || internalExpensifyPath) ? () => Link.openLink(attrHref, environmentURL, isAttachment) : undefined}
```

Comment:
this will cause lint error
```suggestion
            onPress={internalNewExpensifyPath || internalExpensifyPath ? () => Link.openLink(attrHref, environmentURL, isAttachment) : undefined}
```